rasa/validator.py

```diff
@@ -249,8 +252,6 @@ def verify_utterances_in_dialogues(self, ignore_warnings: bool = True) -> bool:
         Checks whether utterances used in the stories are valid,
         and whether all valid utterances are used in stories.
         """
-        everything_is_alright = True
```

Comment:
Why was this removed?

Reply:
My IDE signalled that it wasn't being in use since it's overwritten in L261.